data/notifications.yaml

```diff
@@ -7,14 +7,17 @@
 #   message: |
 #     Markdown message content.
 
-- id: rc-upgrade
-  level: warn
+- id: v2-ga
+  level: note
   scope:
-    - /influxdb/v2.0/
+    - /
   message: |
-    ### InfluxDB OSS 2.0 release candidate
+    ### InfluxDB OSS 2.0 now generally available
```

Comment:
I think an exclamation point would be appropriate. 😇

Reply:
But I'm profligate with exclamation marks.

Reply:
Agreed